virttest/passfd.py

```diff
@@ -39,7 +39,7 @@
 
 import socket
 
-import _passfd
+from . import _passfd
```

Comment:
The Travis ci failed, due to this module is a C-language file, please double check

Reply:
@hsmj1412 updated, thanks.